clients/windows/lockbook/App.xaml.cs

```diff
@@ -2,6 +2,7 @@
 using System.Threading.Tasks;

```

Comment:
This commit message: `dont exit app when clearing state, auto clear state when needed` doesn't seem to be what actually happens -- we don't actually nuke a user's state on `StateRequiresCleaning` do we? If so we should not.